packages/build/tests/functions/tests.js

```diff
@@ -45,7 +45,7 @@ test('Functions: invalid package.json', async (t) => {
   await writeFile(packageJsonPath, '{{}')
   try {
     const output = await new Fixture('./fixtures/functions_package_json_invalid').runWithBuild()
-    t.snapshot(normalizeOutput(output))
+    t.true(output.includes('Configuration error'))
```

Comment:
Can we maybe add this to make the test a bit more accurate?

```suggestion
    t.true(output.includes('Configuration error'))

    // This shape of this error can change with different Node.js versions.
    t.true(
      output.includes('Unexpected token { in JSON at position 1') ||
        output.includes("Expected property name or '}' in JSON at position 1"),
    )    
```

Reply:
@eduardoboucas ah yes! I made a change to your suggestion, where I'll currently be just checking for the one thing these two have in common. `in JSON at position 1`.